### PR TITLE
man: document --prefix option in chage, chpasswd and passwd

### PR DESCRIPTION
**Pull request by jubalh**
Support for `--prefix` was added in
https://github.com/shadow-maint/shadow/pull/714 and is available since shadow 4.14.0.

Close https://github.com/shadow-maint/shadow/issues/822

### CODE REVIEW

File: man/chpasswd.8.xml
Comment (alejandro-colomar) [inline man/chpasswd.8.xml:187]: You may want to use semantic newlines.

```
$ MANWIDTH=72 man man-pages | sed -n '/Use semantic newlines/,/^$/p'
<standard input>:846: warning: table wider than line length minus indentation
   Use semantic newlines
       In the source of a manual page, new sentences should be started
       on  new  lines,  long  sentences  should be split into lines at
       clause breaks (commas, semicolons, colons, and so on), and long
       clauses should be split at phrase boundaries.  This convention,
       sometimes known as "semantic newlines", makes it easier to  see
       the  effect of patches, which often operate at the level of in‐
       dividual sentences, clauses, or phrases.
```

Comment (alejandro-colomar) [reply man/chpasswd.8.xml:187]: This would mean breaking at least after '.', and then maybe also after ':' and at meaningful places.

Comment (jubalh) [reply man/chpasswd.8.xml:187]: Ok :) Will wait for another maintainer to reply and if I should do this change then I will do it for useradd, userdel, usermod, groupadd, groupdel, groupmod as well.

Comment (hallyn) [reply man/chpasswd.8.xml:187]: Thanks - that sounds like it would be nice to have.  Please feel free to open an issue or just a pull request for that.  But of course that's not a part of this change.

Comment (hallyn) [reply man/chpasswd.8.xml:187]: That is - please feel free to make the change to this delta here, but let's not add other manpages, or other chunks of these manpages, to this commit.

Please let me know if you want to squash such a change in this delta.  Otherwise I'll merge as is.

**Review by hallyn — APPROVED** [review]

File: man/chpasswd.8.xml
Comment (jubalh) [reply man/chpasswd.8.xml:187]: @hallyn you can merge as is and I'll create a new PR with the spacing the next days :)

**Pull request merged**
